routes/tasks.ts

```diff
@@ -0,0 +1,78 @@
+import express from "express";
+import DatabaseService from "../services/database.service";
+import { Task } from "../models/task.model";
+
+const router = express.Router();
+let database: DatabaseService;
+
+
+// Initialize database
+async function initializeDatabase() {
+    try {
+        database = new DatabaseService();
+    } catch (error) {
+        console.error("Error connecting to database:", error);
+    }
+}
+
+initializeDatabase().catch(console.error);
+
+
+router.get("/list", async (req: express.Request, res: express.Response) => {
+
+    try {
+        const result = await database.getAllTasks();
+        res.send(result);
+    } catch (error) {
+        console.error("Error retrieving tasks:", error);
+        res.status(500).send("Internal Server Error");
+    }
+});
+router.get("/list/:id", async (req: express.Request, res: express.Response) => {
```

Comment:
would probably be more suitable to do /:id instead of list/:id since youre going through the /tasks endpoint and looking for a specific task not through the list but through all the tasks